data-loading/src/main/java/no/sikt/nva/data/report/api/etl/SingleObjectDataLoader.java

```diff
@@ -27,32 +34,32 @@ public class SingleObjectDataLoader implements RequestHandler<PersistedResourceE
     public static final Logger LOGGER = LoggerFactory.getLogger(SingleObjectDataLoader.class);
     public static final String EXPANDED_RESOURCES_BUCKET = "EXPANDED_RESOURCES_BUCKET";
     public static final String API_HOST = "API_HOST";
+    public static final String EXPORT_BUCKET = "EXPORT_BUCKET";
+    public static final String HYPHEN = "-";
+    public static final String IDENTIFIER = "identifier";
     private final StorageReader<UnixPath> storageReader;
-    GraphService graphService;
+    private final StorageWriter storageWriter;
 
     @JacocoGenerated
     public SingleObjectDataLoader() {
-        this(new GraphService(new GraphStoreProtocolConnection()),
-             new S3StorageReader(new Environment().readEnv(EXPANDED_RESOURCES_BUCKET)));
+        this(new S3StorageReader(new Environment().readEnv(EXPANDED_RESOURCES_BUCKET)),
+             new S3StorageWriter(new Environment().readEnv(EXPORT_BUCKET)));
     }
 
-    public SingleObjectDataLoader(GraphService graphService, StorageReader<UnixPath> storageReader) {
+    public SingleObjectDataLoader(StorageReader<UnixPath> storageReader, StorageWriter storageWriter) {
         LOGGER.info("Initializing SingleObjectDataLoader");
-        this.graphService = graphService;
         this.storageReader = storageReader;
+        this.storageWriter = storageWriter;
     }
 
-    //TODO: Handle failures
     @Override
     public Void handleRequest(PersistedResourceEvent input, Context context) {
         input.validate();
         logInput(input);
         var eventType = EventType.parse(input.eventType());
+        var documentType = DocumentType.fromLocation(input.getLocation());
         if (UPSERT.equals(eventType)) {
-            storeObject(UnixPath.of(input.key()));
-        } else {
```

Comment:
Our resources are "soft deleted", therefore not necessary to listen to delete events in s3

`publication.status` -> `deleted`
 `nviCandidate.applicable` -> `false`